### DISCUSSION TITLE
WIP: doc update

### DISCUSSION THREAD

**Discussion by apatard** [ansible-community/molecule-vagrant#173] [Show and tell]
I've started some time ago to update the documentation, and for now, it looks like: https://github.com/ansible-community/molecule-vagrant/pull/172/.

Before merging, some feedback would be nice, as in:
- what's possibly missing,
- what can be improved in the current PR.

Please share.

**Comment by jimbo8098** [comment]
I'd like to see a newer version of Ubuntu added to the README personally but I know your reasons for not including (discussed in #97 ). I don't think it's wise to rely on CI providers giving access to nested virtualization so I think it's better to just put together a test lib that people can run on their own builds. That kind of relies on trust though. Maybe I could build something in Packer to prove it works on Ubuntu 20.04+?
